### DISCUSSION TITLE
Feature: Update dependencies.

### DISCUSSION THREAD

**Discussion by asova-ebsco** [storybookjs/storybook#14329] [Ideas]
We use a dependency "@storybook/addon-info" in our project.
The "npm audit" shows two Moderate dependency in that module.

Could we update pages:
- "marked"
Path: @storybook/addon-info > marksy > marked (to the version:  >=0.6.2)
- "prismjs"
Path: @storybook/addon-info > @storybook/components >               
                  react-syntax-highlighter > refractor > prismjs (to the version:  >=1.23.0)

**Comment by shilman** [comment]
`addon-info` is deprecated and has been superceded by `addon-docs`, which fixes a bunch of bugs and is easier to maintain. Please give it a try! https://medium.com/storybookjs/storybook-docspage-e185bc3622bf

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-addon-info-addon-notes
